defaults/main.yml

```diff
@@ -21,9 +21,8 @@ ovirt_engine_setup_firewall_manager: 'firewalld'
 
 # This option is suggested from oVirt Documentation
 # https://www.ovirt.org/documentation/install-guide/chap-Installing_oVirt/
-ovirt_engine_setup_update_all_packages: true
 ovirt_engine_setup_update_setup_packages: false
-ovirt_engine_setup_offline: false
+ovirt_engine_setup_offline: "{{ he_offline_deployment | default(false) }}"
```

Comment:
I'm not sure with this `he_offline_deployment` wouldn't it be better to update the `ovirt_engine_setup_offline` with set_facts on your side?